### DISCUSSION TITLE
How to create a People Page with Search option

### DISCUSSION THREAD

**Discussion by adumith** [microsoft-search/pnp-modern-search#2036] [Q&A]
Hello everyone, I'm pretty new to SharePoint, what I learned has been watching videos.

Is there someone who can guide me on how to build my people page with the search option?

The documentation on the page does not help me much.

Thank you in advance.

**Comment by wobba** [comment]
Here's new docs which will be published later https://github.com/microsoft-search/pnp-modern-search/blob/develop/docs/scenarios/Create-a-useful-People-Search.md

**Reply by adumith** [reply]
Thank you very much for answering, 

The filters webpart is not displaying all the fields of the people, such as by location or department.

Additionally, I would like to know how I can customize the way the results are displayed, now it is a vertical line, how can I place it horizontally.

Can be done? How?

Thank in advances,

**Reply by wobba** [reply]
Enter them manually in the combo box and click ENTER.
![image](https://user-images.githubusercontent.com/51104/166411269-d923502d-45c7-4930-be98-6306c3aa6a55.png)

**Comment by adumith** [comment]
Hello @wobba,

Thanks for responding, but when I do that, the filter stops working, is there any way to see that I am receiving information from my users? Maybe I need to make sure of something in my AD. 

What do you think?

Thanks in advance,

![image](https://user-images.githubusercontent.com/101891060/166471816-788d3f5b-0261-4fca-9c28-091d3fa7f333.png)

![image](https://user-images.githubusercontent.com/101891060/166471972-c15f71f7-bfca-4cfe-9ff5-3a87f022a9e2.png)

**Reply by wobba** [reply]
Department comes from the User Profile in SharePoint in this case. You can check what is populated in a profile via SharePoint user profile admin.

**Reply by adumith** [reply]
Hello, Thanks for your reply.

This is interesting, when I go to User Profile in SharePoint I got all the fields, however when I go to edit the filters the department field is not there.

Any idea?

![Picture10](https://user-images.githubusercontent.com/101891060/168333583-11c6e167-5fb0-420f-9b73-54ac76e63bac.png)

**Reply by wobba** [reply]
Type it in manually.
